source/containers.tex

```diff
@@ -2509,7 +2509,7 @@
 &   \requires\ If \tcode{t} is a non-const rvalue expression, \tcode{value_type} shall be
     \tcode{MoveInsertable} into \tcode{X}; otherwise, \tcode{value_type} shall be
     \tcode{CopyInsertable} into \tcode{X}.\br
-    \effects\ Equivalent to a.insert(t).  Return value is an iterator pointing
+    \effects\ Equivalent to \tcode{a.insert(t)}.  Return value is an iterator pointing
```

Comment:
Please remove the trailing backslashes after `\effects` etc., as they are not needed.

Reply:
There are many occurrences of this for many elements in many sources. Is it okay to make a separate PR for that?

Reply:
No, leave the existing document alone. It's just that since you're already touching this part, you might as well clean it up, and I don't want you to introduce new cases of this (in the other change). But don't bother changing other existing instances.

Reply:
Done.